ce/api/util.py

```diff
@@ -248,22 +245,21 @@ def search_for_unique_ids(
     )
 
     matching_cell_methods = filter_by_climatological_statistic(
-        [r[0] for r in cell_methods], climatological_statistic,
+        [r[0] for r in cell_methods],
+        climatological_statistic,
     )
 
     query = (
         sesh.query(mm.DataFile.unique_id)
         .distinct(mm.DataFile.unique_id)
-        .join(
-            mm.DataFileVariableGridded,
-            mm.EnsembleDataFileVariables,
-            mm.Ensemble,
-            mm.Run,
-            mm.Model,
-            mm.Emission,
-            mm.TimeSet,
-            mm.Time,
-        )
+        .join(mm.DataFileVariableGridded)
+        .join(mm.EnsembleDataFileVariables)
+        .join(mm.Ensemble)
+        .join(mm.Run)
+        .join(mm.Model)
+        .join(mm.Emission)
+        .join(mm.TimeSet)
+        .join(mm.Time)
```

Comment:
Good. Are there one or more tests that exerise this code? I'd recommend that there is.

Reply:
Thanks, Rod! Tests are exercising both sections of modified code. The  [first query](https://github.com/pacificclimate/climate-explorer-backend/blob/307a8a87d47049e00042735a4344c52d6c276249/ce/api/util.py#L35C9-L35C10) is tested in [test_utils](https://github.com/pacificclimate/climate-explorer-backend/blob/8247c55c73d516a492d426c96f09b769f88f024c/ce/tests/test_util.py#L127) while the [second query](https://github.com/pacificclimate/climate-explorer-backend/blob/307a8a87d47049e00042735a4344c52d6c276249/ce/api/util.py#L252) is checked during [test_multistats](https://github.com/pacificclimate/climate-explorer-backend/blob/8247c55c73d516a492d426c96f09b769f88f024c/ce/tests/api/test_multistats.py#L45C6-L45C6) and is called from [multistats.py](https://github.com/pacificclimate/climate-explorer-backend/blob/8247c55c73d516a492d426c96f09b769f88f024c/ce/api/multistats.py#L105).